### ISSUE TITLE
Just saying thanks!!!

### ISSUE THREAD

**Issue by ericmjl** [jbarlow83/OCRmyPDF#139]
@jbarlow83, you're amazing for putting this out here. Just wanted to drop a note to say thanks! :smile:

**Comment by ikcalB** [comment]
@jbarlow83 Fully have to agree @ericmjl:

Was trying pdfsandwich before, which was producing enormous file sizes for CANON-brand scanned documents (which does store the text part of the image ccitt encoded).  
Even more, there is only a single option for grayscale / rgb - for the whole document!

ocrmypdf does just use the correct settings (per page) from the original document.  
BIG thumbs up (y)

**Comment by MrReSc** [comment]
@jbarlow83 
I just wanted to say thank you. OCRmyPDF has helped me a lot to realize my paperless office.

**Comment by jbarlow83** [comment]
Thanks everyone :)

**Comment by SpencerRP** [comment]
@jbarlow83 You may want to close this or somehow pin it, if github allows that.

**Comment by wpzdm** [comment]
Yeah, the quality and strength of ocrmypdf is comparable to commercial products. 

Cheers,
Abel

**Comment by arc12** [comment]
+1
Kudos for producing an eminently useful and usable tool... for filling a gap!

**Comment by Eevoo** [comment]
Heyo, +1 here as well. 

Thank you for the excellent software. I was able to plug in my Brother ADS-1200 to Fedora 34 KDE, install the GNOME Document Scanner, save the duplex page as a multi-page PDF, and have OCRmyPDF working in less than 1 hour. 

For Linux on the Desktop, that's an amazing record. 

Great, no, fabulous job on this software package. Thank you!

**Comment by MitanshiKshatriya** [comment]
Thankyou for putting it out here!

**Comment by GoN49** [comment]
+1 You product help a lot and work pretty well on docker. Thanks

**Comment by BillyCroan** [comment]
I offer my kudos as well.  OCRmyPDF is simply outstanding.  The best there is in linux if not anywhere.  Thank you for this invaluable tool!

**Comment by markiv** [comment]
Thank you for this life-enhancing piece of software! 🙏🏽

**Comment by DrewMcArthur** [comment]
chiming in to say thank you for saving me from adobe's bloatware :)

**Comment by NanoExplorer** [comment]
Not only does it do its job adding copyable and searchable text to the pdf, it also reduces its file size! I am very impressed! I do wish that I'd found it before I tried all those other ways of optimizing the file size of my pdf...

**Comment by LondonRain** [comment]
Thanks for the perfectly working software and the **outstanding** documentation. Perfect!

**Comment by realsimix** [comment]
Thank you very much for this great software. It works wonderfully and the documentation is great. And the best, it's open source and well maintained!

**Comment by col-panic** [comment]
One of the best tools on the net! Thank you so much!
